setup.py

```diff
@@ -5,12 +5,14 @@
     description="pybsn is a python interface to Big Switch Networks' products",
     url='https://github.com/floodlight/pybsn',
     author='Jason Parraga, Rich Lane, Andreas Wundsam, Andre Kutzleb',
-    author_email='Sovietaced@gmail.com, Rich.Lane@bigswitch.com, Andreas.Wundsam@bigswitch.com, Andre.Kutzleb@arista.com',
+    author_email='Sovietaced@gmail.com, Rich.Lane@bigswitch.com, Andreas.Wundsam@arista.com, Andre.Kutzleb@arista.com',
     license='ECLIPSE',
     packages=['pybsn'],
     zip_safe=False,
     install_requires=[
-    "requests >= 2.3.0"
+        "requests >= 2.3.0",
```

Comment:
@Andre-Bigswitch FWIW, this commit was added after code review of #30 , and does something that wasn't intended. `pybsn` as a library intentionally did _not_ have `IPython` and `traitlets` as dependencies because they are heavy-weight, and you don't need them if you're just going to use `pybsn` as a library. The deps are only required for `pybsn-repl`.

OTOH, it's of course also inconvenient / can cause frustration to have `pip install pybsn` install `pybsn-repl` but leave it non-functional until you manually install ipython.

I'm wondering how to best express this... maybe the cleanest solution would be to separate out pybsn-repl into its own pypi package? Not sure if it's worth it... @Andre-Bigswitch @ronaldchl  thoughts?

Reply:
> separate out pybsn-repl into its own pypi package
@andi-bigswitch i think that would be worth it. I'd find it interesting to do that myself to learn a bit more about python packaging.